### DISCUSSION TITLE
Sunsetting k8s.gcr.io for new image tags by v1.26.0 release

### DISCUSSION THREAD

**Discussion by puerco** [kubernetes/sig-release#2102] [General]
Since the [release of Kubernetes v1.25](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.25.md#moved-container-registry-service-from-k8sgcrio-to-registryk8sio), the old container registry at [k8s.gcr.io was deprecated](https://groups.google.com/a/kubernetes.io/g/dev/c/DYZYNQ_A6_c/m/oD9_Q8Q9AAAJ) in favor of the new multicloud redirector serving images from registry.k8s.io.

SIG Release is still publishing all new image tags in the old registry and will continue to do so for branches before 1.25. But with the release of 1.26 fast approaching, we are planning the complete sunset of k8s.gcr.io according to the following timeline:

| Date | Relevant Event  | Registry Action |
| --- | --- | --- |
| 6th December 2022 |Release of Kubernetes v1.26 | Tags from 1.25 and 1.26 branches no longer published to k8s.gcr.io |
| 28th February 2023 | EOL of Kubernetes v1.23 | Only patch releases of kubernetes 1.24 go to old registry |
| 28th July 2023 | EOL of Kubernetes v1.24 | No more tags to old registry. |

For more details about the registry, check the [https://registry.k8s.io/ homepage](https://registry.k8s.io/) and the [blog post about backporting the new domain to old branches](https://kubernetes.io/blog/2022/11/28/registry-k8s-io-faster-cheaper-ga/).

As SIG Release is ramping up the implementation of the sunset plan with SIG K8s Infra, we are looking for any considerations or concerns from users, please use this discussion's threads if you want to discuss anything.

**Comment by saschagrunert** [comment]
My only concern is about automated upgrade paths which may begin to fail for 1.25/26 right before the christmas season. I believe this can be solved by communication, so If anyone is aware of more concrete situations please thread in. :upside_down_face: Otherwise LGTM
